src/odemis/acq/stitching/_registrar.py

```diff
@@ -33,6 +33,7 @@
 import numpy
 import math
 from odemis import model
+from odemis.util import almost_equal
```

Comment:
You can drop this import, and this should be fine to merge :-)

Reply:
Done